src/gammatrick.jl

```diff
@@ -175,6 +175,20 @@ function evaluate(H::AbstractPolynomialHomotopy{T}, x::Vector{T}, t::Number) whe
 end
 (H::GammaTrickHomotopy)(x,t) = evaluate(H,x,t)
 
+
+function weylnorm(H::GammaTrickHomotopy{T})  where {T<:Number}
+    f = FP.homogenize.(H.start)
+    g = FP.homogenize.(H.target)
+    λ_1 = FP.weyldot(f,f)
+    λ_2 = real(FP.weyldot(f,g)*H.γ)
+    λ_3 = FP.weyldot(g,g)
+
+    function (t)
+        Float64(sqrt((one(T) - t)^2 * λ_1 + 2 * (one(T) - t) * t * λ_2 + t^2 * abs(H.γ)^2 * λ_3))
```

Comment:
We cannot cast to Float64 here, this may work at the moment but not in the future (if we have higher precision). But I also think the cast should not be necessary.
I think the formula currently has a problem if `t` is `Complex` which happens during the endgame since `(one(T) - t) * t` and `(one(T) - t)^2` would then be again `Complex`. A little bit `real` sprinkled around is necessary I think. Also should the somewhere has to be a complex conjugated `t`?

Reply:
ne because t is real. if we allow complex t then we must also add a couple of absolute values.

Reply:
Then we should restrict t to be real with 
```
function (t::Real)
```

Reply:
And then the `Float64` should just be unnecessary

Reply:
on the other hand there is no reason to not make it available for complex t